src/game_api/math.hpp

```diff
@@ -89,12 +89,114 @@ struct AABB
     float bottom{0};
 };
 
+struct XY
```

Comment:
This should probably be `Vec2` or something like that. No one in the history of linear algebra code called this `XY`  😜
If you want to make sure it doesn't clash with anything else put it into a `Spel2` namespave or something along those lines.

Reply:
where does the '2' come from?

Reply:
It has two elements. As opposed to a `Vec3` which has 3 elements, etc. Same goes for matrices that you'd call `Mat2` and not `XXXYYXYY`

Reply:
btw. i was thinking about constructor and split function from AABB, but not sure which corners i should choose :thinking: